conf/messages.en

```diff
@@ -265,4 +265,9 @@ trusteesDateOfBirth.error.required.two = Date of birth must include a {0} and a
 trusteesDateOfBirth.error.required = Date of birth must include a {0}
 trusteesDateOfBirth.error.invalid = Enter a real date of birth
 trusteesDateOfBirth.error.future = Date of birth must be today or in the past
-trusteesDateOfBirth.error.past = Date of birth must be after 31 December 1499
\ No newline at end of file
+trusteesDateOfBirth.error.past = Date of birth must be after 31 December 1499
+
+trusteeLiveInTheUK.title = Does {0} live in the UK?
+trusteeLiveInTheUK.heading = Does {0} live in the UK?
+trusteeLiveInTheUK.checkYourAnswersLabel = Does {0} live in the UK?
+trusteeLiveInTheUK.error.required = Select yes if the trustee lives in the UK
```

Comment:
The prototype has the following error message for this page 
`Select yes if Adam Conder’s address is in the UK`. Should this become `Select yes if the trustee's address is in the UK' even though it doesn't match the title. Can you ask @R-Derby

Reply:
The content spreadsheet has what you have built currently.

Reply:
Anita is in the middle of writing a ticket to change the errors to what ever entity it is rather than the name after the conversation we had when we found the error